velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -57,18 +65,32 @@ inline Aws::String awsString(const std::string& s) {
   return Aws::String(s.begin(), s.end());
 }
 
-#define VELOX_CHECK_AWS_OUTCOME(outcome, header, bucket, key) \
-  {                                                           \
-    if (!outcome.IsSuccess()) {                               \
-      auto error = outcome.GetError();                        \
-      VELOX_FAIL(                                             \
-          "{} with bucket '{}' and key '{}' due to {}:{}",    \
-          header,                                             \
-          bucket,                                             \
-          key,                                                \
-          error.GetErrorType(),                               \
-          error.GetMessage());                                \
-    }                                                         \
+std::string getErrorStringFromS3Error(
+    const Aws::Client::AWSError<Aws::S3::S3Errors>& error);
+
+inline std::string getS3BackendService(
```

Comment:
This function is not used outside of this file, right? If so, put it in an anonymous namespace.

Reply:
Since the function is tagged inline, will putting it in an anonymous namespace make a difference?